### PR TITLE
Add package and logos

### PR DESCRIPTION
**Pull request by abutterworth**
Adds a package.json file to allow for installation via npm. The intention reflected in this PR is for consumers to include this package this way:

`npm install --save @edx/brand@git+https://git@github.com/edx/brand-edx.org#master`

---

This PR also adds the edx logo and its registered trademark version.

Relates to https://github.com/edx/brand-openedx/pull/1

### CODE REVIEW

**Review by AlasdairSwan — APPROVED** [review]
Looks good :shipit:

File: README.rst [hunk 1]
```diff
@@ -1,6 +1,58 @@
...
+==========
 
 This project contains branding assets and themes for edx.org.
 
-Note: This project should avoid becoming structured as a node module and should have as few dependencies as possible.
+-----
+Usage
+-----
+
+Install this package:
```
Comment (davidjoy) [inline README.rst:10]: As we've started discussing... we ultimately need a better story for how this gets included in MFEs.  For now, do you envision that we'll use our npm aliases mechanism to replace an empty Open edX brand with the edx-specific one?

**Review by davidjoy — APPROVED** [review]
Approving since this is a fine baseline - the Open edX story still needs to be figured out.

File: README.rst (hunk 1 above)
Comment (abutterworth) [reply README.rst:10]: Yes I'm working on an OEP that outlines this.

**Pull request merged**